sdk/avalon_sdk/work_order_receipt/work_order_receipt.py

```diff
@@ -94,7 +98,7 @@ def create_receipt(self, wo_request,
             "receiptCreateStatus": receipt_create_status,
             "workOrderRequestHash": wo_request_hash_str,
             "requesterGeneratedNonce": requester_nonce,
-            "signatureRules":
+            "signaturerules":
```

Comment:
EEA spec refers "signatureRules" in section 7.2.2

Reply:
It was accidental, reverted to "signatureRules"